### PR TITLE
Add zarf

### PR DESCRIPTION
**Pull request by thomasgov**

### CODE REVIEW

File: nifty_pascal-linux/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 FROM alpine AS deploy
 
 # Install build dependencies
-RUN apk add --no-cache afl++ autoconf automake chromium clang clang-analyzer clang-extra-tools cmake curl doxygen file fontconfig gdb g++ git git-lfs libtool linux-headers make msttcorefonts-installer nodejs openssl patch perl py3-colorlog py3-jinja2 py3-lxml py3-setuptools python3 samurai sed util-linux-dev valgrind xz && \
+RUN apk add --no-cache afl++ autoconf automake chromium clang clang-analyzer clang-extra-tools cmake curl doxygen file fontconfig gdb g++ git git-lfs helm libtool linux-headers make msttcorefonts-installer nodejs openssl patch perl podman py3-colorlog py3-jinja2 py3-lxml py3-setuptools python3 samurai sed util-linux-dev valgrind xz && \
```
Comment (nicklauslittle-gov) [inline nifty_pascal-linux/Dockerfile:4]: Have you tested `podman` to make sure it's working? I've been unable to get it to work within docker/k8s. There's some more information about configuring it here: https://wiki.alpinelinux.org/wiki/Podman. But, I'm still running into issues even using that.

Comment (thomasgov) [reply nifty_pascal-linux/Dockerfile:4]: No success, may need to remove podman.

**Comment by thomasgov** [comment]
Update `openjdk17-jdk` to `openjdk21-jdk`

File: nifty_pascal-linux/Dockerfile (hunk 1 above)
Comment (thomasgov) [reply nifty_pascal-linux/Dockerfile]: OBE

**Comment by nicklauslittle-gov** [comment]
I'm good with this. Please squash.

**Pull request merged**